modules/nw-networkpolicy-about.adoc

```diff
@@ -62,6 +62,11 @@ spec:
 
 * Only accept connections from pods within a project:
 +
+[IMPORTANT]
+====
+On an {product-title} cluster that uses the `LoadBalancerService` endpoint publishing strategy, the host network pod on a node cannot communicate with a pod on a different node in the same namespace. This configuration in combination with setting the `allow-from-same-namespace` network policy for your cluster would still result in the same communication limitation.
+====
```

Comment:
Talked to @asood-rh , and we agreed the following note may be more precise:
```
To allow ingress connections from hostNetwork pods in the same namespace, you need to apply "allow-from-hostnetwork" policy together with "allow-same-namespace"
```
Leaving it to the doc pros to rephrase it better :)

Reply:
Thanks, @npinaeva . This note is good. Updated!